zng/resolver/context.go

```diff
@@ -383,16 +415,14 @@ func (c *Context) parseColumn(in string) (string, zng.Column, error) {
 	return rest, zng.NewColumn(name, typ), nil
 }
 
-// parseSetTypeBody parses a set type body of the form "[type]" presuming the set
-// keyword is already matched.
-// The syntax "set[type1,type2,...]" for set-of-arrays is not supported.
-func (c *Context) parseSetTypeBody(in string) (string, zng.Type, error) {
+// parseTypeList parses a type list of the form "[type1,type2,type3]".
+func (c *Context) parseTypeList(in string) (string, []zng.Type, error) {
+	var types []zng.Type
 	rest, ok := match(in, "[")
 	if !ok {
 		return "", nil, zng.ErrTypeSyntax
 	}
 	in = rest
-	var types []zng.Type
```

Comment:
Nit: Maybe keep this here so it stays closer to the first use of `types`.

Reply:
done